### ISSUE TITLE
[Accessibility] [Password Page] [Code] Incorrect HTML language

### ISSUE THREAD

**Issue by ceciliabelem** [cambiatus/frontend#865]
**Describe the bug**
When access muda’s page, the html language should be in portuguese or change as the user changes the navigation language. It impacts the screen readers behaviour. As the page is multilingual, the HTML language should change as the user changes the language.

**To Reproduce**
1. Go to '[Muda's Password](https://muda.cambiatus.io/login?redirect=%2Fdashboard)'
2. Click on 'I'm already a member' button
3. Inspect the HTML 
4. See the incorrect HTML language (in this case, "EN"), even when the user changes the language

**Expected behavior**
If the language is portuguese, the html must be “pt-br” too. In this way, the screen reader verbalizes as native. The same behaviour must work for all languages.

**Screenshots**
![image](https://github.com/user-attachments/assets/f75f8a9a-4345-4b47-ac44-a054a3c73196)

**Cambiatus Env you are using**
Put the URL of the page you are trying to use

**Desktop (please complete the following information):**
 OS: Windows
Browser Chrome
Version 129.0.6668.101 (Versão oficial) 64 bits

**Additional context**

**Comment by ceciliabelem** [comment]
For mapping all defects statistics, I just updated the spreadsheet. Also, there is a graphic to help understand some kinds of bugs and how they can impact users.
[Muda's Report Spreadsheet](https://docs.google.com/spreadsheets/d/11-ld-vQhmtcJhRtPTiUF4YP4bajFFz6U/edit?usp=sharing&ouid=114917425951998070257&rtpof=true&sd=true)
